perun-base/src/main/resources/perun-roles.yml

```diff
@@ -340,17 +355,23 @@ perun_policies:
       - AUDITCONSUMERADMIN:
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   log_String_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   setLastProcessedId_String_int_policy:
```

Comment:
i wouldn't add it in that case, because it feels to me like a result of an internal operation that just updates a counter naturally, as opposed to updating the counter directly by calling the `setLastProcessedId` method. plus we need to enforce MFA globally on these methods, therefore to poll the audit messages one would have to always be authenticated by MFA. what do you think @cuadradek, @xflord ?

Reply:
I agree with @Johaney-s , automatically updating a counter of processed messages doesn't seem like a critical operation.